Dockerfile

```diff
@@ -8,6 +8,10 @@ ARG TARGETVARIANT
 
 # See .dockerignore for the list of files being copied.
 WORKDIR "/src/"
+# add a download step to leverage Docker layer caching
+COPY go.mod go.sum ./
+RUN go mod download
+
 COPY [".", "/src/"]
```

Comment:
How about this?
```suggestion
COPY --exclude=go.mod --exclude=go.sum [".", "/src/"]
```
This may need the following at the beginning of the file:
```
# syntax=docker/dockerfile:1.7-labs

```